newclient/src/core/components/ui/AppStateDiff.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <v-dialog :hide-overlay="true" v-model="showDialog" justify-end width="300" origin="right">
+        <v-btn slot="activator" class="more-btn" color="white" fab depressed>
+            <p class="info--text title pb-2">...</p>
+        </v-btn>
+        <v-card class="overflow-hidden">
+            <v-card-title class="pb-0">
+                <v-flex>
+                    <v-layout row align-center justify-space-between>
+                        <div :class="[titleColor, 'font-weight-bold', 'state-title']">{{ state.status ? status : state.title }}</div>
+                    </v-layout>
+                </v-flex>
+                <v-btn class="close-btn" icon @click="showDialog = false">
+                    <v-icon class="fas fa-times info--text" />
+                </v-btn>
+            </v-card-title>
+            <v-card-text class="pt-2">
```

Comment:
Also can you add class="caption" to the actual text. It will make it smaller

Reply:
done!